src/Symfony/Component/Form/Extension/Core/Type/TextType.php

```diff
@@ -30,13 +32,24 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         }
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function buildView(FormView $view, FormInterface $form, array $options)
+    {
+        if (isset($options['placeholder']) && '' != $options['placeholder']) {
```

Comment:
Strict comparison is more safe.

```suggestion
        if (isset($options['placeholder']) && '' !== $options['placeholder']) {
```